src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonSelectionType.php

```diff
@@ -89,25 +89,23 @@ public function configureOptions(OptionsResolver $resolver)
             ))
         ;
 
-        $resolver->setNormalizers(array(
-            'model_transformer' => function (Options $options, $value) {
-                if (!is_array($value)) {
-                    $value = array(
-                        'class'        => $value,
-                        'save_objects' => true,
-                    );
-                } else {
-                    if (!isset($value['class'])) {
-                        $value['class'] = 'Sylius\Bundle\TaxonomyBundle\Form\DataTransformer\TaxonSelectionToCollectionTransformer';
-                    }
-                    if (!isset($value['save_objects'])) {
-                        $value['save_objects'] = true;
-                    }
+        $resolver->setNormalizer('model_transformer', function (Options $options, $value) {
+            if (!is_array($value)) {
+                $value = array(
+                    'class'        => $value,
+                    'save_objects' => true,
+                );
+            } else {
+                if (!isset($value['class'])) {
+                    $value['class'] = 'Sylius\Bundle\TaxonomyBundle\Form\DataTransformer\TaxonSelectionToCollectionTransformer';
```

Comment:
yeah this is just re-indenting the code.
i thought to fix these class references in a separate PR, and fix all of them at the same time.